pom.xml

```diff
@@ -31,6 +31,9 @@
     <vertx.version>4.3.3</vertx.version>
     <freelib.utils.version>3.1.1</freelib.utils.version>
     <freelib.maven.version>0.3.3</freelib.maven.version>
+    <quartz.version>2.3.2</quartz.version>
```

Comment:
I don't have a preference here, but am curious why you went with quartz instead of one of the other Vert.x native cron / schedulers. It seems to have dependencies (like a JDBC pool, etc.) that we'll be using from Vert.x's related libraries instead of quartz's so wondering if there was some feature it had that we want to use that the others didn't(?)

Reply:
I would like to use Vert.x cron. [They use the Quartz CronExpression format](https://github.com/diabolicallabs/vertx-cron/blob/3eb091703897aa2d628b1c8840af2928ad2de60b/src/main/java/com/diabolicallabs/vertx/cron/CronObservable.java#L35), which is the only reason I've included Quartz as a dep.